docs/pages/admin-guide.mdx

```diff
@@ -1091,19 +1024,19 @@ metadata:
   # connector name that will be used with `tsh --auth=github login`
   name: github
 spec:
-  # client ID of Github OAuth app
+  # Client ID of Github OAuth app
   client_id: <client-id>
-  # client secret of Github OAuth app
+  # Client secret of Github OAuth app
   client_secret: <client-secret>
-  # connector display name that will be shown on web UI login screen
+  # Connector display name that will be shown on web UI login screen
   display: Github
-  # callback URL that will be called after successful authentication
+  # Callback URL that will be called after successful authentication
   redirect_url: https://<proxy-address>/v1/webapi/github/callback
-  # mapping of org/team memberships onto allowed logins and roles
+  # Mapping of org/team memberships onto allowed logins and roles
   teams_to_logins:
     - organization: octocats # Github organization name
       team: admins # Github team name within that organization
-      # allowed logins for users in this org/team
+      # Allowed logins for users in this org/team
       logins:
         - root
```

Comment:
```suggestion
        - access
```
With RBAC in OSS, Github works in a strange way. It's best to define role vs login principle.